mock-identity-system/src/main/java/io/mosip/esignet/mock/identitysystem/service/impl/AuthenticationServiceImpl.java

```diff
@@ -75,6 +82,9 @@ public class AuthenticationServiceImpl implements AuthenticationService {
     @Value("${mosip.esignet.authenticator.auth-factor.kba.field-language}")
```

Comment:
Actually this key should also be named as "mosip.mock.ida.kba.default.field-language"

Reply:
renamed